tests/private-active-active/templates/cloud-config-proxy.yaml

```diff
@@ -37,7 +37,6 @@ write_files:
             refresh_pattern .       0   20% 4320
 
 packages:
-    - https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
     - squid
 
 runcmd:
```

Comment:
Based on our internal version of this setup, we should be able to replace all of these commands with just `service squid reload` 👇🏼